### ISSUE TITLE
Cannot assign bool to property craft\feedme\models\Settings::$parseTwig of type array

### ISSUE THREAD

**Issue by SanboKyo** [craftcms/feed-me#1132]
After upgrading to Craft 4.0.0 ... Feed Me 5.0.1, I get 

```
Exception 'TypeError' with message 'Cannot assign bool to property craft\feedme\models\Settings::$parseTwig of type array'

in /html/craft_w_commerce/vendor/yiisoft/yii2/base/Model.php:741

```
### Additional info

- Craft version: 4.0.0
- PHP version:  8.0.18
- Database version: MySQL 8
- Feed Me version: 5.0.1
- Plugins & versions: ...

**Comment by Metis77** [comment]
While have the same issue, i wonder: is there any workaround for this already

**Comment by Metis77** [comment]
works perfectly, thanks a lot!

**Comment by angrybrad** [comment]
Just released [5.0.2](https://github.com/craftcms/feed-me/releases/tag/5.0.2) with a fixed for this.
